slides/knn/slides-knn.tex

```diff
@@ -42,7 +42,10 @@
 \item $k$-\textbf{NN} can be used for regression and classification.
 \item Generates "similar" predictions for $\xv$ to its $k$ closest neighbors.
 \item "Closeness" requires a distance or similarity measure.
-\item The set containing the $k$ closest points $\xi$ to $\xv$ in the training data is called  the $k$-\textbf{neighborhood} $N_k(\xv)$ of $\xv$.
+\item The subset of the feature space $\Xspace$ that is as least as close to $\xv$ as its $k$-th closest neighbor $\xi[k]$ in the training data set is called the $k$-\textbf{neighborhood} $N_k(\xv)$ of $\xv$:
```

Comment:
- "at least as"
- und es müsste egtl. das subset der Trainingsdaten sein, nicht des feature spaces (bzw. könnte man das diskutieren; aber entweder sind es Beobachtungen => dann aus den Trainingsdaten -- oder ein Bereicht des Feature Spaces => aber dann nicht mit den x_i). Mach das mal mit "subset von D-train" und dann diskutieren wir das noch mal in MM) Danke!

Reply:
habe das jetzt eingefügt, was denkst du? :)